packages/transformers/src/modules/switch_case/switch-case/config.json

```
@@ -0,0 +1,22 @@
+{
+    "label": "Switch Case",
+    "type": "SWITCH_CASE",
+    "class": "SwitchCaseTransformer",
+    "description": "A transformer used to classify a given text to a context label.",
```

Maybe a simpler description?

```suggestion
    "description": "A transformer that outputs the value of a field as its state, provided the field is defined.",
```